.scripts/e2e/testcases.py

```diff
@@ -0,0 +1,70 @@
+import subprocess
+import os
+
+class bcolors:
+    HEADER = '\033[95m'
+    OKBLUE = '\033[94m'
+    OKGREEN = '\033[92m'
+    WARNING = '\033[93m'
+    FAIL = '\033[91m'
+    BOLD = '\033[1m'
+    BLACK = '\033[30m'
+    RED = '\033[31m'
+    GREEN = '\033[32m'
+    YELLOW = '\033[33m'
+    BLUE = '\033[34m'
+    MAGENTA = '\033[35m'
+    CYAN = '\033[36m'
+    WHITE = '\033[37m'
+    UNDERLINE = '\033[4m'
+    ENDC = '\033[0m'
+
+def test_rp_e2e(rp, az, swagger, cliext):
+    print("");
+    logStep ("Start test rp: %s" % rp);
+    cliext_arg_str = '--azure-cli-extension-folder=' + cliext;
+    swagger_arg_str = swagger + '/specification/'+ rp + '/resource-manager/readme.md';
+    az_arg_str = '--use=' + az;
+    logInfo ("Az: " + az_arg_str);
+    logInfo ("Swagger: " + swagger_arg_str);
+    logInfo ("Cliext: " + cliext_arg_str);
+
+    logStep ("Start code generation: %s" % rp);
+    runTask(['autorest', '--version=3.0.6271', '--az', az_arg_str, cliext_arg_str, swagger_arg_str]);
+
+    logStep ("Azdev extension add %s" % rp);
+    runTask(['azdev', 'extension', 'add', rp]);
+
+    logStep ("Azdev extension list");
+    runTask(['azdev', 'extension', 'list', '--query', '[?install==\'Installed\'].{Name:name,Path:path}', '-o', 'table']);
+
+    logStep ("Azdev linter: %s" % rp);
+    runTask(['azdev', 'linter', rp]);
+
+    logStep ("Azdev test: %s" % rp);
+    runTask(['azdev', 'test', rp, '--live']);
```

Comment:
If this RP has not been merged into the azure-cli-extensions repo. you will need --discover when run this azdev test.

Reply:
Not understand, sync offline

Reply:
add --discover